src/uu/expand/src/expand.rs

```diff
@@ -366,10 +366,98 @@ enum CharType {
     Other,
 }
 
-#[allow(clippy::cognitive_complexity)]
-fn expand(options: &Options) -> UResult<()> {
+fn expand_line(
+    buf: &mut Vec<u8>,
+    output: &mut BufWriter<std::io::Stdout>,
+    ts: &[usize],
```

Comment:
Just one nit: because this is now not obvious anymore from context, let's call this `tabstops`.